galata/test/jupyterlab/debugger.test.ts

```diff
@@ -64,6 +64,8 @@ test.describe('Debugger Tests', () => {
     expect(await sourcesPanel.screenshot()).toMatchSnapshot(
       'start-debug-session-sources.png'
     );
+
+    await page.click('jp-button[title^=Continue]');
```

Comment:
I don't know why, but for this specific test we need to continue the execution, otherwise the `afterEach` function is failing.